state_machine/applications/system-check/tests/logging_test.py

```diff
@@ -0,0 +1,86 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+SD Card Logging Test
+* Author(s): Yashika Batra
+"""
+
+import os
+
+
+def sd_test():
+    """
+    Run a basic test; create a file and test existence, write to and
+    read from a file, delete file and test existence. Return result_dict
+    values accordingly
+    """
+
+    # create filepaths
+    filepath = "test.txt"
+    filepath_directory = "/sd/test.txt"
+
+    # try to create a file with the test filepath
+    try:
+        test_file = open(filepath_directory, "x")
+    except OSError:
+        # if the file already exists, remove it and create a new file
+        os.remove(filepath_directory)
+        test_file = open(filepath_directory, "x")
+
+    print("Directory after test.txt was created: {}".format(
```

Comment:
f string

Reply:
probably should have a newline here as well